indexer-app/src/cmr/indexer/data/concepts/collection/community_usage_metrics.clj

```diff
@@ -5,9 +5,16 @@
     [cmr.indexer.data.metrics-fetcher :as metrics-fetcher]))
 
 (def not-provided-version
-  "In EMS community usage CSV, the version value when the version is unknown"
+  "DEPRECATED
+   In EMS community usage CSV, the version value when the version is unknown"
   "N/A")
 
+(defn- valid-version?
```

Comment:
Rename this function and add a doc string. It is not checking if a version is valid or not. It is checking if the collection's version matches the community usage metrics' version.